master/buildbot/status/status_stash.py

```diff
@@ -71,45 +71,76 @@ class StashStatusPush(StatusReceiverMultiService,
                       BuildSetSummaryNotifierMixin):
     implements(IStatusReceiver)
 
-    def __init__(self, base_url, user, password):
+    def __init__(self, base_url, user, password,
+                 key_format='%(builderName)s',
```

Comment:
why do you want to have such a complicated implementation?  you use `builder.render` for various things below, why not use it to **just** render the `key_format` (and `name_format`) parameter?